package/kedro_viz/services/layers_services.py

```diff
@@ -0,0 +1,125 @@
+# Copyright 2021 QuantumBlack Visual Analytics Limited
```

Comment:
does the filename need `_services` tacked on if it belongs in `services/`? Maybe just `services/layers.py`?